org.metaborg.spoofax.shell.core/src/main/java/org/metaborg/spoofax/shell/output/PrintResult.java

```diff
@@ -0,0 +1,67 @@
+/**
+ *
+ */
+package org.metaborg.spoofax.shell.output;
+
+import java.util.Collections;
+import java.util.List;
+
+import org.metaborg.core.source.ISourceRegion;
+
+/**
+ * A result that represents output that is human readable.
+ */
+public class PrintResult implements IResult {
```

Comment:
Since this class seems to depend on the Fold regions, would it not be wise to make a common ancestor/somehow denote this? They both have a `List<ISourceRegion>` which is meant to be the same fold regions, right?